HousingSearchListener/V1/Interfaces/HttpClientHandler.cs

```diff
@@ -0,0 +1,33 @@
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace HousingSearchListener.V1.Interfaces
+{
+    public class HttpClientHandler : IHttpHandler
+    {
+        private HttpClient _client = new HttpClient();
```

Comment:
HttpClient should only be created once per process then re-used.
You should inject the IHttpClientFactory and use that (see my PersonApi gateway class in the tenure listener)

Reply:
I went that approach initially.
I wanted to stub the call to the PersonApi for E2E though and I couldn't find a way for the HttpClientFactory to return an interface (none exist for HttpClient).
It's only being created one. It's a singleton injection.